gitlabform/gitlab/branches.py

```diff
@@ -67,50 +67,6 @@ def get_branch_access_levels(self, project_and_group_name, branch):
             "projects/%s/protected_branches/%s", (project_and_group_name, branch)
         )
 
-    def get_only_branch_access_levels(self, project_and_group_name, branch):
```

Comment:
Moved to a helper in `tests/acceptance/__init__.py` as it is only used for asserts there.